### PR TITLE
update the publications of each cell death type

### PR DESCRIPTION
**Pull request by MikeLippincott**
Update the publication stats of each cell death type

### CODE REVIEW

File: figures/S14/data/apoptosis_results.csv [hunk 1]
```diff
@@ -1,37 +1,38 @@
 Search query: apoptosis
```
Comment (axiomcura) [inline figures/S14/data/apoptosis_results.csv:1]: Have you done anything to the data collection to refine your search? Earlier it seems like they got an extra publication, I don't know if you're aware of that. (e.g.2011, 2009, 2005, 2004 and 1993)

File: figures/S14/data/autophagy_results.csv [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+2016,5510
+2015,5016
 2014,4338
-2013,3688
+2013,3687
 2012,3023
 2011,2316
-2010,1874
+2010,1872
 2009,1442
```
Comment (axiomcura) [inline figures/S14/data/autophagy_results.csv:19]: Similar to my comment above, but in this case, each previous year lost one publication.

File: figures/S14/data/necrosis_results.csv [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
+2023,25722
+2022,28549
+2021,28339
+2020,28420
+2019,26035
+2018,25889
+2017,25468
 2016,25658
-2015,26561
+2015,26560
```
Comment (axiomcura) [inline figures/S14/data/necrosis_results.csv:14]: Similar comment as above

File: figures/S14/figures/cell_death_over_time.png
Comment (axiomcura) [inline figures/S14/figures/cell_death_over_time.png:1]: Looking at your plot code, did you mean 2024?

**Review by axiomcura — APPROVED** [review]
LGTM! Left some comments c:

File: figures/S14/data/apoptosis_results.csv (hunk 1 above)
Comment (MikeLippincott) [reply figures/S14/data/apoptosis_results.csv:1]: Who is they and what extra publication?

File: figures/S14/data/autophagy_results.csv (hunk 2 above)
Comment (MikeLippincott) [reply figures/S14/data/autophagy_results.csv:19]: insteresting...no explaination here

File: figures/S14/figures/cell_death_over_time.png
Comment (MikeLippincott) [reply figures/S14/figures/cell_death_over_time.png:1]: Yes, changed and pushed

**Pull request merged**